core/storage/user/gae_models.py

```diff
@@ -164,13 +165,33 @@ def export_data(user_id):
             'role': user.role,
             'username': user.username,
             'normalized_username': user.normalized_username,
-            'last_agreed_to_terms': user.last_agreed_to_terms,
+            'last_agreed_to_terms': (
+                time.mktime(user.last_agreed_to_terms.timetuple())
```

Comment:
Use a method in utils.py, this is too low-level and shouldn't be done directly in the code here -- the repetition of the same logic again and again is not so good. Ditto below.

Reply:
Done